frontend/src/routes/Map.jsx

```diff
@@ -189,7 +189,7 @@ const Map = () => {
         </ul>
       </ReactMapGL>
       <input
-        className="pl-4 bg-transparent border-solid border-2 border-white rounded-full h-11 mt-5 focus:outline-none text-slate-200"
+        className="pl-4 bg-transparent border-solid border-2 border-white rounded-full h-11 mt-5 focus:outline-none text-slate-200 absolute z-50 bottom-10 bg-red-400 transform -translate-x-1/2 left-1/2"
```

Comment:
For future devs wondering.  According to ChatGPT:


To center an element using Tailwind CSS, you can use Flexbox utilities to horizontally center an absolute-positioned element within its container.

You can achieve this by adding left-1/2 to move the left edge of the element to the center of the container and transform -translate-x-1/2 to shift it back by half its own width.

Here's an example of how you can center your element:

```html
<div class="relative flex justify-center">
  <button class="pl-4 bg-transparent border-solid border-2 border-white rounded-full h-11 mt-5 focus:outline-none text-slate-200 absolute z-50 bottom-10 bg-red-400 left-1/2 transform -translate-x-1/2">
    Your Button
  </button>
</div>
```

This code applies left-1/2 to the button to horizontally center it within its parent container using Flexbox. The transform -translate-x-1/2 class then shifts it back by half its width, making it perfectly centered horizontally.

Adjust the button's content and other styles as needed, but maintaining the left-1/2 and transform -translate-x-1/2 classes for horizontal centering.